src/loaddb/load_session.cpp

```diff
@@ -242,13 +248,68 @@ namespace cubload
 
 	// notify session that batch is done
 	m_session.notify_batch_done (m_batch.get_id ());
+
+	thread_ref.m_loaddb_driver->clear ();
       }
 
     private:
       const batch &m_batch;
       session &m_session;
+      css_conn_entry &m_conn_entry;
   };
 
+  static loaddb_worker_context_manager *g_loaddb_wp_context_manager;
+  static cubthread::entry_workpool *g_loaddb_worker_pool;
+  static std::mutex g_loaddb_wp_mutex;
+  static unsigned int g_loaddb_session_count = 0;
+
+  void load_wp_register_session ()
+  {
+    g_loaddb_wp_mutex.lock ();
+
+    if (g_loaddb_session_count == 0)
+      {
+	assert (g_loaddb_worker_pool == NULL);
+	assert (g_loaddb_wp_context_manager == NULL);
+
+	unsigned int pool_size = prm_get_integer_value (PRM_ID_LOADDB_WORKER_COUNT);
+	std::string worker_pool_name ("loaddb-workers");
```

Comment:
I think you can pass directly "loaddb-workers" string to create_worker_pool function (worker_pool_name was used before to append a session id, but now it no longer the case)